tests/formats/test_brfld.py

```diff
@@ -28,3 +28,60 @@ def test_dread_brfld_100(dread_tree_100, brfld_path):
 @pytest.mark.parametrize("brfld_path", bossrush_assets)
 def test_dread_brfld_210(dread_tree_210, brfld_path):
     parse_build_compare_editor(Brfld, dread_tree_210, brfld_path)
+
+
+def test_get_actors_methods(dread_tree_100):
+    scenario = dread_tree_100.get_file("maps/levels/c10_samus/s010_cave/s010_cave.brfld", Brfld)
+
+    actors_for_sublayer_names = []
+
+    for sublayer in scenario.sublayers_for_actor_layer():
+        actors_for_sublayer_names += scenario.actors_for_sublayer(sublayer).keys()
+
+    all_actors_in_actor_layer_names = [
+        actor_name for sublayer_name, actor_name, actor in scenario.all_actors_in_actor_layer()
+    ]
+
+    assert actors_for_sublayer_names == all_actors_in_actor_layer_names
+
+
+def test_follow_link(dread_tree_100):
+    scenario = dread_tree_100.get_file("maps/levels/c10_samus/s010_cave/s010_cave.brfld", Brfld)
+
+    actor_link = scenario.link_for_actor("cubemap_fr.2_cave_ini", "cubes", ActorLayer.LIGHTS)
+
+    assert scenario.follow_link(actor_link).sName == "cubemap_fr.2_cave_ini"
+
+
+def test_remove_actor_from_actor_group(dread_tree_100):
```

Comment:
this test as well as `test_add_actor_to_actor_groups` should be parametrized instead of hardcoding three cases for each